Scripts/summaryStatistics/biophysics/aspect_ratio.py

```diff
@@ -0,0 +1,79 @@
+# Standard modules
+import sys
+import numpy as np
+import pickle #only needed for testing
+
+# Installed modules 
+import CellModeller
+
+# Local/custom modules
+from lje.src.lownerJohnEllipse import plot_ellipse, welzl
+
+def get_centroids(cells):    
+    """
+    Obtain cell controids
+    @param  cells           cellStates dict
+    @return centroids       nparray of centroids (n_cells, 2)
+    """                
+    # Initialize storage variables
+    n_cells = len(cells.keys())
+    centroids = np.zeros((n_cells, 2))
+    
+    # Collect centroid values from cellStates dict
+    for i, cell in enumerate(cells.values()):
+        centroids[i][0] = cell.pos[0]
+        centroids[i][1] = cell.pos[1]
+    
+    return centroids    
+     
+def calc_aspect_ratio(a, b):
+    """
+    Compute the aspect ratio of an ellipse
+
+    @param  a               (float) one axis of the ellipse
+    @param  b               (float) other axis of the ellipse
+    @return aspect_ratio    (float) short/long axis
+    """
+    aspect_ratio = np.minimum(a, b)/np.maximum(a, b)
+
+    return aspect_ratio
+
+def main(cells):
```

Comment:
the main function of the modules is not called "main". Please change this to whatever you want but not "main".